src/Select/__stories__/Select.stories.js

```diff
@@ -23,25 +23,31 @@ export default {
 };
 
 export const primary = () => (
-    <Select
-        options={options}
-        placeholder='Select' />
+    <>
+        <Select
+            aria-label='Primary'
+            options={options}
+            placeholder='Select' />
+    </>
 );
 
 export const compact = () => (
     <Select
+        aria-label='Compact'
         compact
         options={options}
         placeholder='Select' />
 );
 export const disabled = () => (
     <Select
+        aria-label='Disabled'
         disabled
         options={options}
         placeholder='Select' />
 );
 export const readOnly = () => (
     <Select
+        aria-label='readOnly'
```

Comment:
Wasn't sure how the screen reader would sound with the concatenated word `readOnly` but it sounds no different than if it were "Read Only" 👍